// [[Reason: Sourcemap is more detailed]] ////

//// [tests/cases/compiler/flatArrayNoExcessiveStackDepth.ts] ////

===================================================================
--- TSC	declarations
+++ DTE	declarations
@@ -1,6 +1,6 @@
 
 //// [flatArrayNoExcessiveStackDepth.d.ts.map]
-{"version":3,"file":"flatArrayNoExcessiveStackDepth.d.ts","sourceRoot":"","sources":["flatArrayNoExcessiveStackDepth.ts"],"names":[],"mappings":"AAEA,OAAO,CAAC,MAAM,GAAG,EAAE,OAAO,EAAE,CAAC;AAC7B,QAAA,MAAM,GAAG,EAAE,MAAM,EAAmC,CAAC;AAErD,UAAU,GAAI,SAAQ,KAAK,CAAC,MAAM,CAAC;CAAG;AAItC,QAAA,MAAM,WAAW,UAAW,OAAO,KAAG,IAMrC,CAAC;AAEF,iBAAS,CAAC,CAAC,GAAG,EAAE,CAAC,SAAS,MAAM,EAAE,CAAC,EAAE,SAAS,CAAC,GAAG,EAAE,GAAG,CAAC,EAAE,CAAC,EAAE,SAAS,CAAC,GAAG,EAAE,CAAC,CAAC,GAAG,IAAI,CAGpF"}
+{"version":3,"file":"flatArrayNoExcessiveStackDepth.d.ts","sourceRoot":"","sources":["flatArrayNoExcessiveStackDepth.ts"],"names":[],"mappings":"AAEA,OAAO,CAAC,MAAM,GAAG,EAAE,OAAO,EAAE,CAAC;AAC7B,QAAA,MAAM,GAAG,EAAE,MAAM,EAAmC,CAAC;AAErD,UAAU,GAAI,SAAQ,KAAK,CAAC,MAAM,CAAC;CAAG;AAItC,QAAA,MAAM,WAAW,GAAI,KAAK,EAAE,OAAO,KAAG,IAMrC,CAAC;AAEF,iBAAS,CAAC,CAAC,GAAG,EAAE,CAAC,SAAS,MAAM,EAAE,CAAC,EAAE,SAAS,CAAC,GAAG,EAAE,GAAG,CAAC,EAAE,CAAC,EAAE,SAAS,CAAC,GAAG,EAAE,CAAC,CAAC,GAAG,IAAI,CAGpF"}
 
-//// https://sokra.github.io/source-map-visualization#base64,ZGVjbGFyZSBjb25zdCBmb286IHVua25vd25bXTsNCmRlY2xhcmUgY29uc3QgYmFyOiBzdHJpbmdbXTsNCmludGVyZmFjZSBGb28gZXh0ZW5kcyBBcnJheTxzdHJpbmc+IHsNCn0NCmRlY2xhcmUgY29uc3QgcmVwcm9fNDMyNDk6ICh2YWx1ZTogdW5rbm93bikgPT4gdm9pZDsNCmRlY2xhcmUgZnVuY3Rpb24gZjxBcnIsIEQgZXh0ZW5kcyBudW1iZXI+KHg6IEZsYXRBcnJheTxBcnIsIGFueT4sIHk6IEZsYXRBcnJheTxBcnIsIEQ+KTogdm9pZDsNCi8vIyBzb3VyY2VNYXBwaW5nVVJMPWZsYXRBcnJheU5vRXhjZXNzaXZlU3RhY2tEZXB0aC5kLnRzLm1hcA==,eyJ2ZXJzaW9uIjozLCJmaWxlIjoiZmxhdEFycmF5Tm9FeGNlc3NpdmVTdGFja0RlcHRoLmQudHMiLCJzb3VyY2VSb290IjoiIiwic291cmNlcyI6WyJmbGF0QXJyYXlOb0V4Y2Vzc2l2ZVN0YWNrRGVwdGgudHMiXSwibmFtZXMiOltdLCJtYXBwaW5ncyI6IkFBRUEsT0FBTyxDQUFDLE1BQU0sR0FBRyxFQUFFLE9BQU8sRUFBRSxDQUFDO0FBQzdCLFFBQUEsTUFBTSxHQUFHLEVBQUUsTUFBTSxFQUFtQyxDQUFDO0FBRXJELFVBQVUsR0FBSSxTQUFRLEtBQUssQ0FBQyxNQUFNLENBQUM7Q0FBRztBQUl0QyxRQUFBLE1BQU0sV0FBVyxVQUFXLE9BQU8sS0FBRyxJQU1yQyxDQUFDO0FBRUYsaUJBQVMsQ0FBQyxDQUFDLEdBQUcsRUFBRSxDQUFDLFNBQVMsTUFBTSxFQUFFLENBQUMsRUFBRSxTQUFTLENBQUMsR0FBRyxFQUFFLEdBQUcsQ0FBQyxFQUFFLENBQUMsRUFBRSxTQUFTLENBQUMsR0FBRyxFQUFFLENBQUMsQ0FBQyxHQUFHLElBQUksQ0FHcEYifQ==,Ly8gUmVwcm8gZnJvbSAjNDM0OTMKCmRlY2xhcmUgY29uc3QgZm9vOiB1bmtub3duW107CmNvbnN0IGJhcjogc3RyaW5nW10gPSBmb28uZmxhdE1hcChiYXIgPT4gYmFyIGFzIEZvbyk7CgppbnRlcmZhY2UgRm9vIGV4dGVuZHMgQXJyYXk8c3RyaW5nPiB7fQoKLy8gUmVwcm9zIGZyb20gY29tbWVudHMgaW4gIzQzMjQ5Cgpjb25zdCByZXByb180MzI0OSA9ICh2YWx1ZTogdW5rbm93bik6IHZvaWQgPT4gewogICAgaWYgKHR5cGVvZiB2YWx1ZSAhPT0gInN0cmluZyIpIHsKICAgICAgICB0aHJvdyBuZXcgRXJyb3IoIk5vIik7CiAgICB9CiAgICBjb25zdCBtYXRjaCA9IHZhbHVlLm1hdGNoKC9hbnl0aGluZy8pIHx8IFtdOwogICAgY29uc3QgWywgZXh0cmFjdGVkXSA9IG1hdGNoOwp9OwoKZnVuY3Rpb24gZjxBcnIsIEQgZXh0ZW5kcyBudW1iZXI+KHg6IEZsYXRBcnJheTxBcnIsIGFueT4sIHk6IEZsYXRBcnJheTxBcnIsIEQ+KTogdm9pZCB7CiAgICB4ID0geTsKICAgIHkgPSB4OyAgLy8gRXJyb3IKfQo=
+//// https://sokra.github.io/source-map-visualization#base64,ZGVjbGFyZSBjb25zdCBmb286IHVua25vd25bXTsNCmRlY2xhcmUgY29uc3QgYmFyOiBzdHJpbmdbXTsNCmludGVyZmFjZSBGb28gZXh0ZW5kcyBBcnJheTxzdHJpbmc+IHsNCn0NCmRlY2xhcmUgY29uc3QgcmVwcm9fNDMyNDk6ICh2YWx1ZTogdW5rbm93bikgPT4gdm9pZDsNCmRlY2xhcmUgZnVuY3Rpb24gZjxBcnIsIEQgZXh0ZW5kcyBudW1iZXI+KHg6IEZsYXRBcnJheTxBcnIsIGFueT4sIHk6IEZsYXRBcnJheTxBcnIsIEQ+KTogdm9pZDsNCi8vIyBzb3VyY2VNYXBwaW5nVVJMPWZsYXRBcnJheU5vRXhjZXNzaXZlU3RhY2tEZXB0aC5kLnRzLm1hcA==,eyJ2ZXJzaW9uIjozLCJmaWxlIjoiZmxhdEFycmF5Tm9FeGNlc3NpdmVTdGFja0RlcHRoLmQudHMiLCJzb3VyY2VSb290IjoiIiwic291cmNlcyI6WyJmbGF0QXJyYXlOb0V4Y2Vzc2l2ZVN0YWNrRGVwdGgudHMiXSwibmFtZXMiOltdLCJtYXBwaW5ncyI6IkFBRUEsT0FBTyxDQUFDLE1BQU0sR0FBRyxFQUFFLE9BQU8sRUFBRSxDQUFDO0FBQzdCLFFBQUEsTUFBTSxHQUFHLEVBQUUsTUFBTSxFQUFtQyxDQUFDO0FBRXJELFVBQVUsR0FBSSxTQUFRLEtBQUssQ0FBQyxNQUFNLENBQUM7Q0FBRztBQUl0QyxRQUFBLE1BQU0sV0FBVyxHQUFJLEtBQUssRUFBRSxPQUFPLEtBQUcsSUFNckMsQ0FBQztBQUVGLGlCQUFTLENBQUMsQ0FBQyxHQUFHLEVBQUUsQ0FBQyxTQUFTLE1BQU0sRUFBRSxDQUFDLEVBQUUsU0FBUyxDQUFDLEdBQUcsRUFBRSxHQUFHLENBQUMsRUFBRSxDQUFDLEVBQUUsU0FBUyxDQUFDLEdBQUcsRUFBRSxDQUFDLENBQUMsR0FBRyxJQUFJLENBR3BGIn0=,Ly8gUmVwcm8gZnJvbSAjNDM0OTMKCmRlY2xhcmUgY29uc3QgZm9vOiB1bmtub3duW107CmNvbnN0IGJhcjogc3RyaW5nW10gPSBmb28uZmxhdE1hcChiYXIgPT4gYmFyIGFzIEZvbyk7CgppbnRlcmZhY2UgRm9vIGV4dGVuZHMgQXJyYXk8c3RyaW5nPiB7fQoKLy8gUmVwcm9zIGZyb20gY29tbWVudHMgaW4gIzQzMjQ5Cgpjb25zdCByZXByb180MzI0OSA9ICh2YWx1ZTogdW5rbm93bik6IHZvaWQgPT4gewogICAgaWYgKHR5cGVvZiB2YWx1ZSAhPT0gInN0cmluZyIpIHsKICAgICAgICB0aHJvdyBuZXcgRXJyb3IoIk5vIik7CiAgICB9CiAgICBjb25zdCBtYXRjaCA9IHZhbHVlLm1hdGNoKC9hbnl0aGluZy8pIHx8IFtdOwogICAgY29uc3QgWywgZXh0cmFjdGVkXSA9IG1hdGNoOwp9OwoKZnVuY3Rpb24gZjxBcnIsIEQgZXh0ZW5kcyBudW1iZXI+KHg6IEZsYXRBcnJheTxBcnIsIGFueT4sIHk6IEZsYXRBcnJheTxBcnIsIEQ+KTogdm9pZCB7CiAgICB4ID0geTsKICAgIHkgPSB4OyAgLy8gRXJyb3IKfQo=
 
